### ISSUE TITLE
Suggestion / ideas for cinematic mode . .

### ISSUE THREAD

**Issue by charlesmlamb** [godotengine/godot-proposals#2665]
### Describe the project you are working on
A Tomb raider game . .

### Describe the problem or limitation you are having in your project
Well, it's not a problem, but I've been thinking, about a nice ' cinematic ' preview mode, for Godot . . I've been thinking, sometimes life is nice, or you have a really nice day, and everything looks beautiful, or stunning, like ' better ' . . it's almost like God's blessed you, with something 'extra' nice or, not sure . . . What comes closest to those moments, of stunning clarity, and ' serenity ', even insight . . is classic, gorgerous, stylized nature photography, like here . . 

![2021-04-28 0932](https://user-images.githubusercontent.com/83387915/116504144-fa7e6100-a8b7-11eb-99c5-a0c277f3d722.jpg)

You also sometimes get ' a ' feeling, when looking at photo, of rain-forest animals . .

![2021-04-28 0937](https://user-images.githubusercontent.com/83387915/116504175-0b2ed700-a8b8-11eb-836c-92f2f908fb85.jpg)

Also, like this . .

![2021-04-28 0953](https://user-images.githubusercontent.com/83387915/116504205-13871200-a8b8-11eb-927b-efb72a1b73d8.jpg)

Or, sometimes architecture photography, has that ' look ', like things couldn't be better, lovely day, just ' perfect ' . .

![2021-04-28 0954](https://user-images.githubusercontent.com/83387915/116504241-2699e200-a8b8-11eb-9640-17aafcadb864.jpg)

Maybe that could be the ' cinematic preview ', in Godot :O :O . .

I found these pictures, by looking for prize-winning nature photography or, prize-winning architecture photography, there are also youtube videos, about the technique, or so . . It can even be about people or, more, same ' look ' . . Not sure how they do it, but moments where they capture the beauty of nature, or ' Heaven ' <3

![2021-04-29 0701](https://user-images.githubusercontent.com/83387915/116504480-a6c04780-a8b8-11eb-9129-1e6c8749955f.jpg)

Well, it's an idea . . it's as beautiful, as the world can look, to the human eyes, and ' soul ', like how we see things, when everything is just ' right ' and, amazing . . 

### Describe the feature / enhancement and how it helps to overcome the problem or limitation
Well, I was thinking Godot might want a cinematic preview or, even this could ' inspire ' the default viewport, though it should prob. just be clean, ordinary . . anyway, nature documentaries, or so . . can look really good, it's like a subtle photo-enhancement, that brings out the beauty, in every-thing, even grass or, leaves . . or, dirt or, an old coat, or so . .

"Nature photography preview mode", or " classic architecture / design preview mode " . . 

Sometimes, a really nice, 'calm', well-taken photo, of the carnivals in Rio <3 has, same effect . . it's ' right ', you want to be there and, every-one is happy and, feeling nice, better . . . <3 <3 . .

![2021-04-28 0958](https://user-images.githubusercontent.com/83387915/116504735-51386a80-a8b9-11eb-9704-7cfc2074ab6a.jpg)

'Perfect' moment, of clarity and, insight, or so . . Anyway, I suppose that's how God, or the angels see the world, in ' beauty ', clarity, detail, ' crispness ' . . . .

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams
Well, it's just a proposal, for a nice ' cinematic preview ' mode, that takes it a ' bit ' higher, the visual fidelity and, control . . .

### If this enhancement will not be used often, can it be worked around with a few lines of script?
I'm not sure, it's more about finding a ' nice ' look, or ' ideas ' for a ' cinematic preview ' :OO . . .

### Is there a reason why this should be core and not an add-on in the asset library?
Again, not sure, I never get this, 9 / 10 I been wrong :(( :(( <3 <3 . . . .

**Comment by charlesmlamb** [comment]
ps. I made a new e-mail, bec. I wish to publish under an alias, my games, since I don't like ' celebrity ' or, over - much attention . . I tend to make really long, and well, very long suggestions, or so . . had to make a new gitHub as well, want to be anonymous, that's all . . .

Another example . .

![2021-04-29 0720](https://user-images.githubusercontent.com/83387915/116505646-b1301080-a8bb-11eb-8ae5-950d9a70a3ac.jpg)

Architecture photos sometimes have it, it's most often seen in ' nature ' documentaries, maybe could call it ' HRD light ', or 'muted ' HRD, mixed with contrast and, saturation modifications, maybe hue and, tone . .

![2021-04-28 0956](https://user-images.githubusercontent.com/83387915/116505697-da50a100-a8bb-11eb-8f26-9276f28ec0ef.jpg)

Sometimes, photos of people are like that too, not ' dramatic ', of fake, just beautiful, calm, amazing and, ' Holy ', blessed . . . .

![2021-04-28 1009](https://user-images.githubusercontent.com/83387915/116505721-f05e6180-a8bb-11eb-8a6e-391cd95ea3b3.jpg)

![2021-04-29 0725](https://user-images.githubusercontent.com/83387915/116505781-108e2080-a8bc-11eb-91f4-1d8bda51549e.jpg)

It's difficult to find the ' right ' photo, after gitHub scaling, so on . . also, other people have different monitors, but classic nature photography, or ' nature ' documentaries, high - quality, TEND to have them, the ' expensive ' stuff . . .

<3 . . .

**Comment by charlesmlamb** [comment]
So, last . . If you are looking for the sweet, spot, between ' no ' effects and, Hollywood . . That's prob. it . . .

Also, you could get a PBR rainforest stage, use it in trailers for demos, ' content ' and, put it in that mode or, so :OO 

![2021-04-29 0837](https://user-images.githubusercontent.com/83387915/116511067-2bfe2900-a8c6-11eb-8ef9-d87760cf2e1c.jpg)

So, the boring, mundane, looks ' better ', holy . . . .

![2021-04-29 0838](https://user-images.githubusercontent.com/83387915/116511131-46d09d80-a8c6-11eb-88d6-186b7d7720b4.jpg)

Or, . . 

![2021-04-29 0841](https://user-images.githubusercontent.com/83387915/116511455-c199b880-a8c6-11eb-8353-0bbd8bc4f4ed.jpg)

You get it, sometimes ' nature ' and, the world is nice, to ' look ', study, ' understand ' <3 . .

**Comment by charlesmlamb** [comment]
Sorry, imo this would give you an 'enhanced' mode, what the ' big, corrupt ' medie calls ' Cinematic mode ' and, would look better than any engine, currently and, should give people the ' visuals ' to make amazing ' drama ' or, fiction . .

Also, have a ' standard ' to match PBR, reflections, fluid sims, a TON more, to . .

Classic, gorgeous, pristine nature ' documentary ' or, graphics . . .

Thanks <3 . .

Last proposal, now I make games and, thank you, for this amazing, fast, impressive little engine, can't wait <3 Too, awe-some . . . .

**Comment by SIsilicon** [comment]
Could you summarize all that? I have no clue what

**Comment by charlesmlamb** [comment]
Yes, some-one said there would be a ' cinematic preview ', I suppose it means, not sure . .

well, that things look ' cinematic ', or not sure, :(( . .

my idea was, maybe make ' cinematic ' preview look, like this, since it's quite nice and, well rather nice or, so . .

similar, to ACES, or well, not sure . .

this would give ' Godot ' a Paradise, un-spoiled look, that ' makes ' sense, not sure . . .

Or, like ' nature ' . . . <3

**Comment by SIsilicon** [comment]
The proposal sounds way too vague. Do you have any idea his the controls would be like on the user's end (non rhetorically)?

**Comment by SIsilicon** [comment]
And "cinematic preview" does indeed exist, but I think you don't understand what it does. If you have say, a bunch of cameras in a scene, and an animation player was used to cut between the cameras (change the current camera), then enabling cinematic preview allows you to view the animation from the perspective of the current camera the animation cuts to.

**Comment by charlesmlamb** [comment]
wait, ' Cinematic preview ' is not like post - proc :O . .

I thought, it was close, to that . . .

**Comment by SIsilicon** [comment]
Nope, not at all. All it does is let you preview the scene through its cameras better. I used it in a game jam before when editing a cutscene. I was cutting through multiple cameras so it proved to be very useful in that situation.

This video explains and shows the cinematic preview in action.

**Comment by charlesmlamb** [comment]
Wow, I didn't get it, sorry . . .

I'm not sure what it is, but it's not ' effects ' <3 :OO . . . .

My, bad . . .

**Comment by Zireael07** [comment]
@charlesmlamb: 'Cinematic preview' is just that, preview (mostly for animations), it was never supposed to be about 'effects'
